src/accountsdb/buffer_pool.zig

```diff
@@ -19,26 +21,75 @@ comptime {
     _ = offset;
 }
 
+const FrameRef = packed struct(u32) {
+    f_idx: FrameIndex,
```

Comment:
`index`? feels like the frame part is already infered from the struct name?

Reply:
renamed to index f40689214b7f8a0945d24c5a2847c29af6db0203